hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CreateTableProcedure.java

```diff
@@ -285,6 +290,13 @@ private void preCreate(final MasterProcedureEnv env)
           getTableName(), (newRegions != null ? newRegions.size() : 0));
     }
 
+    if(StringUtils.isEmpty(tableDescriptor.getValue(TRACK_IMPL))){
```

Comment:
Seems the formatter file in your IDE is bit strange? Usually we will have a space between 'if' and '('